.github/workflows/ci.yml

```diff
@@ -0,0 +1,105 @@
+name: linkedevents-ci
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+env:
+  POSTGRES_DB: test_linkedevents
+  POSTGRES_USER: postgres
+  POSTGRES_PASSWORD: postgres
+  DATABASE_URL: "postgis://postgres:postgres@localhost/test_linkedevents"
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [ 3.9 ]
+    services:
+      postgres:
+        image: helsinkitest/postgis:9.6-2.5-alpine
+        env:
+          # Note: not redundant, parent env is not applied to service
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test_linkedevents
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: psycopg2 prerequisites
+        run: sudo apt-get install libpq-dev
+
+      - name: System requirements for Django
+        run: sudo apt-get install gdal-bin gettext postgresql-client
+
+      - name: Setup postgres extensions
```

Comment:
I wonder if we really this step. :thinking: There are Django migrations in place which at least in theory should load `hstore` and `pg_trgm` since we're not skipping migrations (running `pytest` with `--no-migrations`).

Reply:
Oh, looks like it works fine without.  Whoops.